### DISCUSSION TITLE
Keyframe not found (smartcut)

### DISCUSSION THREAD

**Discussion by 1777777789** [mifi/lossless-cut#1977] [Support]
Hi,
Really appreciate this program even though sometimes gives weird errors like this one I got.
Apparently it stops the process of saving cuts and doesn't proceed to merge them if it encounters one cut without any keyframe.
Is it possible to encode the whole cut without a keyframe or maybe use the previous keyframe ? 
Usually the cut and merge works however this last cut was at the end of the video and had no keyframes in it.
Thank you very much

**Comment by 1777777789** [comment]
I'm running 3.59.1.
The error code Is: "Cannot find any keyframe after the desired start cut point".  I noticed that there ARE keyframes after my cut point. Maybe this happens because the keyframe is too far from the cutpoint? (example of one that was 32 seconds after, but still inside the segment). When I place the starting cut point right BEFORE the previous keyframe this gets fixed, however the cut is not precise anymore as I would expect with a smartcut.
I find myself having to go back to each segment with this error quite a few times over the exporting process, and place the start cut right before a previous keyframe.

**Comment by mifi** [comment]
> Maybe this happens because the keyframe is too far from the cutpoint? (example of one that was 32 seconds after, but still inside the segment).

it will search for up to 60 seconds after the cutpoint, so this shouldn't be the reason.

Maybe related #1190

There's discussions about improving smart cut here: #126

if you can provide a (small) file that reproduces the problem, i can look into it
